src/frontend/src/lib/components/send/QRCodeScan.svelte

```diff
@@ -81,3 +81,9 @@
 		{$i18n.core.text.back}
 	</button>
 </ButtonGroup>
+
+<style>
+	.qr-code-wrapper {
+		color: rgba(0, 0, 0, 0.6) !important;
```

Comment:
Please avoid using `!important` as it will inevitably lead to problems.

We can for example use following here probably `--primary-rgb: 0, 0, 0`

Or maybe something like `--primary-rgb: 50, 20, 105` if we want to use a color of the theme

Reply:
changed to use `--primary-rgb: 50, 20, 105`